contracts/token_abi/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Mohammad"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "token_abi"
+
+[dependencies]
+std = { git = "https://github.com/FuelLabs/sway", branch = "master" }
```

Comment:
You can and probably should remove the explicit `std` dependency here, so that you can use the tagged release version automatically.

Reply:
Wait you didn't remove this one :joy:

Reply:
WaT 🤪